docs/source/_templates/docs-sidebar.html

```diff
@@ -1,7 +1,4 @@
 
-<a class="navbar-brand" href="{{ pathto(master_doc) }}">
```

Comment:
Here is a proposal to get the logo back : https://github.com/apache/arrow-datafusion/pull/5688